mmv1/third_party/terraform/services/container/resource_container_cluster_test.go.erb

```diff
@@ -1553,7 +1548,7 @@ func TestAccContainerCluster_withNodeConfigKubeletConfigSettings(t *testing.T) {
 		CheckDestroy: testAccCheckContainerClusterDestroyProducer(t),
 		Steps: []resource.TestStep{
 			{
-				Config: testAccContainerCluster_withNodeConfigKubeletConfigSettings(clusterName, networkName, subnetworkName),
+				Config: testAccContainerCluster_withNodeConfigKubeletConfigSettingsBaseline(clusterName, networkName, subnetworkName),
```

Comment:
I tried also adding a 
```
resource.TestCheckNoResourceAttr("google_container_cluster.with_node_config_kubelet_config_settings",
						"node_config.0.kubelet_config.0.cpu_cfs_quota"),
```
etc, and it didn't work -- I think it works as expected, so I just removed that block for now.